README.md

```diff
@@ -276,7 +276,7 @@ conda install -c conda-forge numpyro
    Your options are:
 
    - Call the distribution directly and provide a `PRNGKey`, e.g. `dist.Normal(0, 1).sample(PRNGKey(0))`
-   - Provide the `rng_key` argument to `numpyro.sample`. e.g. `numpyro.sample('x', dist.Normal(0, 1), rng_key=PRNGKey(0))`.
+   - Provide the `key` argument to `numpyro.sample`. e.g. `numpyro.sample('x', dist.Normal(0, 1), key=PRNGKey(0))`.
```

Comment:
I think `rng_key` is correct.

Reply:
https://github.com/pyro-ppl/numpyro/blob/56b88c3e46f9572d400b82cf34920975b55b7e88/numpyro/distributions/distribution.py#L296

Reply:
The signature is at the `__call__` method: https://github.com/pyro-ppl/numpyro/blob/56b88c3e46f9572d400b82cf34920975b55b7e88/numpyro/distributions/distribution.py#L358

Reply:
the readme is demoing `sample` not `__call__`:
```
>>> dist.Normal(0, 1)(rng_key=jax.random.PRNGKey(0))
Array(-0.20584226, dtype=float32)
>>> dist.Normal(0, 1).sample(key=jax.random.PRNGKey(0))
Array(-0.20584226, dtype=float32)
>>> dist.Normal(0, 1).sample(rng_key=jax.random.PRNGKey(0))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: sample() got an unexpected keyword argument 'rng_key'
```

Reply:
oh gotcha got confused about what the readme was demo'ing